Le débug OAuth2 cause un crash applicatif

```
GET /login/openid - req.headers.referer: https://stylo.huma-num.fr/books
/authorization-code/callback - callback InternalOAuthError: failed to obtain access token
    at /usr/src/app/node_modules/passport-openidconnect/lib/strategy.js:93:38
    at /usr/src/app/node_modules/oauth/lib/oauth2.js:191:18
    at passBackControl (/usr/src/app/node_modules/oauth/lib/oauth2.js:132:9)
    at IncomingMessage.<anonymous> (/usr/src/app/node_modules/oauth/lib/oauth2.js:157:7)
    at IncomingMessage.emit (events.js:326:22)
    at endReadableNT (_stream_readable.js:1252:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21) {
  oauthError: {
    statusCode: 400,
    data: '<html><head><title>Bad request</title>\n' +
      '<style>\n' +
      'body{background:#000;color:#fff;padding:10px 50px;font-family:sans-serif;}a{text-decoration:none;color:#fff;}h1{text-align:center;}\n' +
      '</style>\n' +
      '</head>\n' +
      '<body>\n' +
      '<h1>Bad request</h1>\n' +
      '<p>invalid_request</p>\n' +
      '<center><a href="https://lemonldap-ng.org">LemonLDAP::NG</a></center>\n' +
      '</body>\n' +
      '</html>'
  }
} undefined undefined
```

```
/usr/src/app/app.js:239
        console.error('Unable to authenticate', info.message)
                                                     ^

TypeError: Cannot read property 'message' of undefined
    at /usr/src/app/app.js:239:54
    at Strategy.strategy.error (/usr/src/app/node_modules/passport/lib/middleware/authenticate.js:358:18)
    at /usr/src/app/node_modules/passport-openidconnect/lib/strategy.js:93:32
    at /usr/src/app/node_modules/oauth/lib/oauth2.js:191:18
    at passBackControl (/usr/src/app/node_modules/oauth/lib/oauth2.js:132:9)
    at IncomingMessage.<anonymous> (/usr/src/app/node_modules/oauth/lib/oauth2.js:157:7)
    at IncomingMessage.emit (events.js:326:22)
    at endReadableNT (_stream_readable.js:1252:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! graphql-stylo@0.1.0 prod: `NODE_ENV=production node --heapsnapshot-signal=SIGUSR2 app.js`
npm ERR! Exit status 1
```

--- a/graphql/app.js
+++ b/graphql/app.js
@@ -235,15 +235,18 @@ app.use('/authorization-code/callback',
       failureFlash: true
     }, (err, user, info) => {
       console.log('/authorization-code/callback - callback', err, user, info)
-      if (!user) {
-        console.error('Unable to authenticate', info.message)
-        res.redirect(`/error?message=${info.message}`)
-      } else if (err) {
-        console.error('error', err)
-        res.redirect(`/error?message=${info.message}`)
-      } else {
+      if (user) {
         req.user = user
         next()
+      } else {
+        const errorMessages = []
+        if (info && info.message) {
+          errorMessages.push(info.message)
+        }
+        if (err && err.message) {
+          errorMessages.push(err.message)
+        }
+        res.redirect(`/error?message=${errorMessages.join(' ')}`)
       }
     })(req, res, next)
   },

